server/routerlicious/packages/services-ordering-rdkafka/src/rdkafkaBase.ts

```diff
@@ -22,6 +23,7 @@ export interface IKafkaEndpoints {
 export abstract class RdkafkaBase extends EventEmitter {
     protected readonly kafka: typeof kafkaTypes;
     protected readonly sslOptions?: kafkaTypes.ConsumerGlobalConfig;
+    protected defaultRestartOnKafkaErrorCodes: number[] = [];
```

Comment:
Could `readonly` be added to this? It might be since it's only set in the constructor

Reply:
Unfortunately it seems that making it readonly does not allow me to set the value in the child class constructor:

![Screen Shot 2021-12-03 at 1 19 07 PM](https://user-images.githubusercontent.com/41453887/144674227-22b44c5d-d5ee-4ccb-9926-984857270aab.png)

Any alternative ideas?

Reply:
I don't know of any alternative. It's not a big deal though so it's fine as it is.

Reply:
Cool. I will merge then